hooks/pre-commit

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#
+# Enure everything is fine before committing stuff
+#
+
+git secrets --pre_commit_hook -- "$@"
```

Comment:
@pp-ps here we need to define a set of regex

Reply:
They are in a dedicated "provisioner" file.